scripts/stablex/setup_thegraph_data.js

```diff
@@ -0,0 +1,122 @@
+const { solutionSubmissionParams, basicTrade } = require("../../test/resources/examples")
+
+const { makeDeposits, placeOrders } = require("../../test/stablex/stablex_utils")
+const {
+  closeAuction,
+  getOwl,
+  getBatchExchange,
+  setAllowances,
+  addTokens,
+  deleteOrders,
+  submitSolution,
+  getBatchId,
+  createMintableToken,
+  mintTokens,
+  mintOwl,
+} = require("../../scripts/stablex/utilities.js")
+
+const { toETH } = require("../../test/resources/math")
+const { waitForNSeconds } = require("../../test/utilities")
+
+module.exports = async function(callback) {
+  try {
+    const batchExchange = await getBatchExchange(artifacts)
+    const owl = await getOwl(artifacts)
+
+    // Prepare user accounts
+    const accounts = await web3.eth.getAccounts()
+    const [user1Address, user2Address] = accounts
+    const usersAddresses = [user1Address, user2Address]
```

Comment:
Why this? userAddress and accounts are the same thing no?

Reply:
you could condense lines `27, 28, 29` via `const [user1Address, user2Address] = await web3.eth.getAccounts()`

Reply:
Good question. It looks the same, but is not :)

accounts has all the web3 accounts. This is for ganache so you'll see 10 accounts there.
I'm using destructuring to have a slice of that array while I create some consts for the individual  addresses that I will need later